sncosmo/builtins.py

```diff
@@ -750,6 +750,152 @@ def load_2011fe(relpath, name=None, version=None):
     _SOURCES.register_loader(name, load_timeseries_ascii,
                              args=(relpath,), version='2.0', meta=meta)
 
+# V19
+V19_CC_models = [('v19_asassn14jb_corr', '1.0','V19_HostExtCorr', 'SN II', 'V19_ASASSN14jb_HostExtCorr.SED'),
+                ('v19_asassn14jb', '1.0', 'V19_noHostExtCorr', 'SN II', 'V19_ASASSN14jb_noHostExtCorr.SED'),
+                ('v19_asassn15oz_corr', '1.0','V19_HostExtCorr', 'SN II', 'V19_ASASSN15oz_HostExtCorr.SED'),
```

Comment:
All existing sources use hyphens in their names rather than underscores, so it would best to keep that consistency. So `v19-asassn15oz-corr`.